translations/README-ge.md

```diff
@@ -0,0 +1,137 @@
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3202/badge)](https://bestpractices.coreinfrastructure.org/projects/3202)
+[![BCH compliance](https://bettercodehub.com/edge/badge/litmuschaos/litmus?branch=master)](https://bettercodehub.com/)
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Flitmuschaos%2Flitmus.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Flitmuschaos%2Flitmus?ref=badge_shield)
+[![YouTube Channel](https://img.shields.io/badge/YouTube-Subscribe-red)](https://www.youtube.com/channel/UCa57PMqmz_j0wnteRa9nCaw)
+<br><br><br><br>
+
+#### *Das README in [anderen Sprachen](translations/TRANSLATIONS.md).*
+
+[🇰🇷](translations/README-ko.md) [🇨🇳](translations/README-chn.md)
+
+## Überblick
+
+Litmus ist ein Werkzeug für cloud-natives Chaos Engineering. Die Anwendung stellt
+verschiedene Hilfmittel bereit Chaos auf Kubernetes zu inszenieren und hilft
+somit SREs Schwachstellen in ihren Entwicklungen zu finden.
+Mit Litmus können Chaos-Experimente in der Staging-Umgebung und gegebenenfalls
+auch in der Produktion durchgeführt werden um Schwachstellen und Bugs zu
+finden.
+
+Litmus nutzt einen cloud-nativen Ansatz um Chaos zu erzeugen, verwalten und zu
+beobachten. Um Chaos zu initialisieren und manipulieren werden die folgenden
+Kubernetes Custom Resource Definitions (kurz CRDs) benötigt:
+- **ChaosEngine**: Mittel, um eine Kubernetes-Anwendung der eine
+  Kubernetes-Node mit dem Chaos-Experiment zu verbinden. Der Chaos-Engine wird
+  vom Chaos-Operator beobachtet, welcher die Chaos-Experimente ausführt.
+- **ChaosExperiment**: Mittel, um die Konfigurationsparameter eines
+  Chaos-Experiments zu gruppieren. Chaos-Experiment CRs werden vom Operator
+  kreiert wenn Experimente vom Chaos-Enginge ausgeführt werden.
+- **ChaosResult**: Mittel, um die Ergebnisse des Chaos-Experiments zu
+  speichern. Der Chaos-Exporter liest die Ergebnisse aus und exportiert die
+  Messwerte auf einen konfigurierten Prometheus-Server.
+
+Die Chaos-Experimente werden auf <a href="https://hub.litmuschaos.io" target="_blank">hub.litmuschaos.io</a> gehostet. Das ist die zentrale Stelle, an der die Entwicker von Anwendungen oder die Nutzer ihre Chaos-Experimente teilen, sodass diese genutzt werden können um die Belastbarkeit und Sicherheit der Anwendung im Endeinsatz weiter zu verbessern. 
+
+![Litmus workflow](/images/litmus-arch_1.png)
+
+## Anwendung
+
+- **Für Entwickler**: Mit Litmus können Chaos-Experimente in der
+  Entwicklungsphase einer Anwendung als Erweiterung zu Modultests
+  oder auch Integrationstests durchgeführt werden.
+- **Für CI-pipelines**: In der kontinuierlichen Integration können
+  Chaos-Experimente als Schritt in der Pipeline genutzt werden um Bugs zu
+  finden.
+  - **Für SREs**: Mit Litmus können Chaos-Experimente geplant durchgeführt
+    werden um möglichst früh Schwächen in Anwendungen zu finden und die
+    Stabilität des Endproduktes zu steigern.
```

Comment:
This won't be a sub-point, please revert it back as the main point in this list.
```suggestion
- **Für SREs**: Mit Litmus können Chaos-Experimente geplant durchgeführt
  werden um möglichst früh Schwächen in Anwendungen zu finden und die
  Stabilität des Endproduktes zu steigern.
```